Nexus.Sdk.Token/TokenServerProvider.cs

```diff
@@ -610,6 +610,24 @@ public async Task<CustomerDataResponse> GetCustomerData(string customerCode)
             return await builder.ExecuteGet<CustomerDataResponse>();
         }
 
+        /// <summary>
+        /// Get customer trace based on the code
```

Comment:
Please update documentation/comments to map what is being done in method

Reply:
[Update doc comments for Customer Trace endpoint](https://github.com/QuantozTechnology/Nexus.Sdk.Dotnet/pull/46/commits/5028eba0005f38889bce8839fdca5c3094e85b10)